### PR TITLE
added events page and other minor edits

### PR DESCRIPTION
**Pull request by microlei**

### CODE REVIEW

File: docs/about/index.md
Comment (nathanweeks) [inline docs/about/index.md:1]: This page is auto-generated from templates/about_template.md; suggest removing it from the repository.

File: docs/resources/index.md
Comment (nathanweeks) [inline docs/resources/index.md:1]: This one too...

**Comment by microlei** [comment]
Should we also remove the tags pages from the repo?

**Comment by nathanweeks** [comment]
> Should we also remove the tags pages from the repo?

Yes. There's a gitignore in the tags directory (https://github.com/harvardinformatics/informatics-website/tree/main/docs/resources/tags), but it looks like one .md file must have been pushed in another branch before that was merged.

**Pull request merged**